src/pages/index.js

```diff
@@ -68,10 +102,59 @@ editProfileButton.addEventListener("click", () => {
   editProfilePopup.open();
 });
 
+const handleDeleteClick = (card) => {
+  confirmationPopup.open();
+  confirmationPopup.setSubmitCallback(() => {
+    confirmationPopup.setButtonText(true, "Deleting...");
+
+    api
+      .deleteCard(card._cardId)
+      .then(() => {
+        card.delete();
+      })
+      .catch((err) => {
+        console.error(err);
+      })
+      .finally(() => {
+        confirmationPopup.close();
+      });
+  });
+};
+
+const handleLikeClick = (card) => {
+  if (card._isLiked) {
```

Comment:
`_isLiked`

* Please, delete the **low dash** from methods if you want to use them `outside` a class because the `low dash` shows a **private** method which is used only inside the class.